src/components/joinus/JoinUs.md

```diff
@@ -0,0 +1,23 @@
+# Junte-se a nós!
+
+Um dos objetivos principais do OpenDevUFCG é ajudar os projetos open source feitos e apoiados por estudantes de Ciência da Computação da UFCG a atrair contribuições.
+
+### Tem um projeto open source e deseja impulsioná-lo?
+
+O **IssueAi** tem o intuito de ser um espaço que promove a ajuda e colaboração de alunos para alunos através da exposição dos seus respectivos projetos para a comunidade da UFCG, fazendo crescer o espírito open source dentro do nosso ambiente acadêmico.
+
+Para ajudar ainda mais a atingir tal objetivo, orientamos que todo projeto tenha:
+
+-   Um bom README. É importante tentar fazer as pessoas não terem medo da primeira contribuição que é a mais importante de todas, por isso seja claro, simples e amigável, é a chave para atrair contribuições.
+
+-   Uma licença Open Source.
+
+-   Código de conduta. É preciso garantir um ambiente acolhedor e respeitável para todos.
+
+-   Guia de contribuição. Uma boa documentação dos passos a serem seguidores para o uso do projeto tem que ser prioridade. Tente ser claro e direto, use o auxílio de outras documentações como hyperlinks.
+
+Para se aprofundar ainda mais no assunto, recomenda-se a leitura do [open source guide](https://opensource.guide/starting-a-project/).
+
+### Para adicionar um projeto
+
+Os dados dos repositórios usados para mostrar são recuperados a partir da [API do Laguinho](https://github.com/OpenDevUFCG/laguinho-api). Para adicionar seu repositório aos nossos projetos apoiados, confira o passo-a-passo [aqui](https://github.com/OpenDevUFCG/laguinho-api/blob/master/docs/REPOSITORIES.md#como-contribuir-adicionando-mais-reposit%C3%B3rios).
```

Comment:
In my last pr, (#81), i update this to get from here, so please update the message. Sorry for this confusion, i should warn you